sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -360,7 +360,8 @@ export class KeyClient {
       const unflattenedProperties = {
         enabled: options.enabled,
         notBefore: options.notBefore,
-        expires: options.expiresOn
+        expires: options.expiresOn,
```

Comment:
Shouldn't this be `expiresOn` as well?

Reply:
no, this is what gets sent to the service